Xamarin.Forms.Platform.UAP/TabbedPageRenderer.cs

```diff
@@ -249,20 +252,33 @@ void OnLoaded(object sender, RoutedEventArgs args)
 				case NotifyCollectionChangedAction.Replace:
 					if (e.NewItems != null)
 						for (int i = 0; i< e.NewItems.Count; i++)
-							(e.NewItems[i] as Page).PropertyChanged += OnChildPagePropertyChanged;
+							((Page)e.NewItems[i]).PropertyChanged += OnChildPagePropertyChanged;
```

Comment:
Why change these?

Reply:
`InvalidCastException` seemed both more informative and less verbose.